mono/mini/exceptions-arm64.c

```diff
@@ -97,7 +97,7 @@ mono_arch_get_call_filter (MonoTrampInfo **info, gboolean aot)
 	offset += num_fregs * 8;
 	ctx_offset = offset;
 	ctx_offset += 8;
```

Comment:
Nit: I think the more correct change (in the sense of saving a couple of bytes on the stack frame) would be to change this line:
```c
ctx_offset += 8;
```
to
```c
offset += 8;
```
then the frame size computation below would be correct.

Great find! 👍

Reply:
We should totally do this. The code can be pretty confusing as is. Having a mysterious unused stack slot makes it also harder to follow.